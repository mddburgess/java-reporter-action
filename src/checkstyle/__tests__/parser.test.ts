import CheckstyleParser from "../parser";

describe("CheckstyleParser", () => {
  it("can parse an empty Checkstyle report", () => {
    const report = new CheckstyleParser("src/checkstyle/__tests__/__fixtures__/empty.xml").read();
    expect(report).toMatchSnapshot();
  });
});
