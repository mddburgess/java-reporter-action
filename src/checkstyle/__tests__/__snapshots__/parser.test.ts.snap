// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckstyleParser can parse an empty Checkstyle report 1`] = `
Object {
  "violations": Array [],
}
`;
